frontend/src/pages/register.tsx

```diff
@@ -40,46 +41,56 @@ export default function Register() {
   };
 
   return (
-    <Container component="main" maxWidth="xs">
-      <header style={{ textAlign: "center", margin: "20px 0" }}>
-        <Link to="/" style={{ fontSize: "24px", fontWeight: "bold", color: "#1D4ED8", textDecoration: "none" }}>
-          Pro Tube
-        </Link>
-      </header>
-      <main>
-        <Typography variant="h5" align="center">Register</Typography>
-        <form onSubmit={handleSubmit}>
-          <TextField
-            margin="normal"
-            required
-            fullWidth
-            id="username"
-            label="Username"
-            autoComplete="username"
-            autoFocus
-            value={username}
-            onChange={(e) => setUsername(e.target.value)}
-          />
-          <TextField
-            margin="normal"
-            required
-            fullWidth
-            id="password"
-            label="Password"
-            type="password"
-            autoComplete="new-password"
-            value={password}
-            onChange={(e) => setPassword(e.target.value)}
-          />
-          {error && <Typography color="error" align="center">{error}</Typography>}
-          <Button type="submit" variant="contained" color="primary" fullWidth>
+    <Box sx={{ backgroundColor: '#1c1c1e', minHeight: '100vh', display: 'flex', alignItems: 'center', justifyContent: 'center' }}>
+      <Container component="main" maxWidth="xs" sx={{ backgroundColor: '#2e2e2e', padding: 4, borderRadius: 2, boxShadow: 3 }}>
+        <header style={{ textAlign: "center", marginBottom: "20px" }}>
+          <Link to="/" style={{ fontSize: "24px", fontWeight: "bold", color: "#FF0000", textDecoration: "none", textShadow: '1px 1px 2px #FFFFFF, -1px -1px 2px #FFFFFF, 1px -1px 2px #FFFFFF, -1px 1px 2px #FFFFFF' }}>
+            Pro Tube
+          </Link>
+        </header>
+        <main>
+          <Typography variant="h5" align="center" color="#FFFFFF" sx={{ marginBottom: 2 }}>
             Register
-          </Button>
-          <p style={{ textAlign: "center", marginTop: "10px" }}>
-            Already have an account? <Link to="/login" style={{ color: "#1D4ED8", textDecoration: "none" }}>Login</Link>
-          </p>
-        </form>
-      </main>
-    </Container>
+          </Typography>
```

Comment:
Ya tenéis otro PR abierto con estos mismos cambios, que además no tienen nada que ver con el objetivo de este en concreto